### ISSUE TITLE
Avoid using predictions.txt in Monolix reader

### ISSUE THREAD

**Issue by baltcir1** [ggPMXdevelopment/ggPMX#124]
At the time of alpha ggPMX testing by Monolix SME team, I made a comment that prediction.txt should be replaced by the corresponding output under chart data folder. Motivation is to increase compatibility if merging is required between several outputs under chart data (rounding is consistent here) and because prediction.txt is an old and historical file… I don't know if Lixoft will maintain it. Hope that helps.

**Comment by agstudy** [comment]
This implies : 

 - Change to predictions section in  configuration file : ggPMX/inst/templates/mlx18/standing.ipmx
 - Create a new function like read_mlx_pred to read  the new predictions file

Hope that helps

**Comment by baltcir1** [comment]
Merci Amine! 👍

**Comment by biethbr1** [comment]
@baltcir1: can you precisely identified the new output in the chart data folder?

**Comment by baltcir1** [comment]
Yes, for Monolix 2018, I think its in ChartsData/ScatterPlotOfTheResiduals/xxx_residuals.txt where "xxx" is the name of the output variable (not 100% sure if it changes between projects). If we change to the above, then, we'll have a difference between 2016 and 2018. But there was a good reason for changing and in the long run, most users will use recent Monolix versions anyways.
